dcos/subcommand.py

```diff
@@ -686,6 +653,69 @@ def package_json(self):
             return util.load_json(package_json_file)
 
 
+class Subproc():
```

Comment:
We should move this out of the `subcommand` module. Are we using this for _all_ exec/fork now?